docs/embedding/signed-embedding-parameters.md

```diff
@@ -121,6 +119,10 @@ For example, the following embedding URL will display an embedded item in dark m
 your_embedding_url#theme=night&titled=true&bordered=false
 ```
 
+¹ Available on paid plans.
```

Comment:
Can we move these footnotes to directly beneath the tables? And link to [https://www.metabase.com/pricing](https://www.metabase.com/pricing)?